config/initializers/refresh_external_reference_unique_id.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+if ActiveRecord::Base.connection.table_exists?("external_references")
+  ExternalReference.pluck(:external_type).uniq.each do |type|
+    ExternalReference.where(external_type: type).find_each do |entry|
+      puts type
+      refreshed_unique_id = entry.external.unique_id
```

Comment:
warum überhaupt `entry`, es ist an der stelle eine `external_reference`, oder?

ich würde mich, und in der zukunft sicher jeder von uns, hier auch über ein kommentar freuen, um festzuhalten, was und warum hier passiert.

Reply:
because it's shorter ;)